contributors/devel/coding-conventions.md

```diff
@@ -1,138 +1 @@
-
-  - Significant features should come with integration (test/integration) and/or
-[end-to-end (test/e2e) tests](e2e-tests.md)
-    - Including new kubectl commands and major features of existing commands
-
-  - Unit tests must pass on OS X and Windows platforms - if you use Linux
-specific features, your test case must either be skipped on windows or compiled
-out (skipped is better when running Linux specific commands, compiled out is
-required when your code does not compile on Windows).
-
-  - Avoid relying on Docker hub (e.g. pull from Docker hub). Use gcr.io instead.
-
-  - Avoid waiting for a short amount of time (or without waiting) and expect an
-asynchronous thing to happen (e.g. wait for 1 seconds and expect a Pod to be
-running). Wait and retry instead.
-
-  - See the [testing guide](testing.md) for additional testing advice.
-
-## Directory and file conventions
-
-  - Avoid package sprawl. Find an appropriate subdirectory for new packages.
-(See [#4851](http://issues.k8s.io/4851) for discussion.)
-    - Libraries with no more appropriate home belong in new package
-subdirectories of pkg/util
-
-  - Avoid general utility packages. Packages called "util" are suspect. Instead,
-derive a name that describes your desired function. For example, the utility
-functions dealing with waiting for operations are in the "wait" package and
-include functionality like Poll. So the full name is wait.Poll
-
-  - All filenames should be lowercase
-
-  - Go source files and directories use underscores, not dashes
-    - Package directories should generally avoid using separators as much as
-possible (when packages are multiple words, they usually should be in nested
-subdirectories).
-
-  - Document directories and filenames should use dashes rather than underscores
-
-  - Contrived examples that illustrate system features belong in
-/docs/user-guide or /docs/admin, depending on whether it is a feature primarily
-intended for users that deploy applications or cluster administrators,
-respectively. Actual application examples belong in /examples.
-    - Examples should also illustrate [best practices for configuration and
-using the system](https://kubernetes.io/docs/user-guide/config-best-practices/)
-
-  - Third-party code
-
-    - Go code for normal third-party dependencies is managed using
-[Godeps](https://github.com/tools/godep)
-
-    - Other third-party code belongs in `/third_party`
-      - forked third party Go code goes in `/third_party/forked`
-      - forked _golang stdlib_ code goes in `/third_party/golang`
-
-    - Third-party code must include licenses
-
-    - This includes modified third-party code and excerpts, as well
-
-## Coding advice
-
-  - Go
-
-    - [Go landmines](https://gist.github.com/lavalamp/4bd23295a9f32706a48f)
+This document has been moved to https://git.k8s.io/community/contributors/guide/coding-conventions.md
```

Comment:
This link doesn't work - could it be because of the redirect stuff, and it would work once /coding-conventions lives in /guide?

Reply:
If this is a content tombstone, please drop a note for when it's ok to remove, eg: https://github.com/kubernetes/community/blob/master/contributors/devel/release/README.md